packages/core/src/virtual-entrypoint.ts

```diff
@@ -312,9 +312,18 @@ function shouldSplitRoute(routeName: string, splitAtRoutes: (RegExp | string)[]
   }
   return splitAtRoutes.find(pattern => {
     if (typeof pattern === 'string') {
+      if (pattern.startsWith('/') && pattern.endsWith('/')) {
+        let fragments = pattern.match(/\/(.*?)\/([a-z]*)?$/i);
+
+        if (!fragments) {
+          throw new Error(`Unable to parse splitAtRoutes pattern ${pattern}`);
+        }
+        let regex = new RegExp(fragments[1], fragments[2] || '');
+        return regex.test(routeName);
+      }
       return pattern === routeName;
     } else {
-      return pattern.test(routeName);
+      throw new Error('Invalid splitAtRoutes type found');
```

Comment:
I'm not sure this is useful or valid 🤔 I was toying with just removing the surrounding `if (typeof pattern === 'string') {` since they will always be serialised into a string so the check is pointless. 

Any thoughts?

Reply:
I moved the fix down to the deserialisation layer so this code didn't need to change any more